producer-kafka/README.adoc

```diff
@@ -0,0 +1,407 @@
+= Using the Kafka CDC Producer
+
+This extension provides support for sending Change Data Capture(CDC) events to an Apache Kafka®.
+It is designed to work with the Stargate.
+
+== Kafka CDC - Stargate integration
+
+This extension is designed to integrate with Stargate. It uses an link:https://www.osgi.org/[OSGi] framework for registering the extension within Stargate.
+The `KafkaProducerActivator` class handles the registration process. When you put the `producer-kafka-{version}.jar` in the
```

Comment:
We need to introduce a way to allow registration based on config, otherwise we can't have it in tree.

Reply:
Yes, I agree. The dynamic registration will be also important when setting configuration from UI and allowing clients to restart only given extension. But I don't see how it relates to this documentation. 
Do you want me to remove the paragraph about putting a jar in `stargate-lib`? It seems that this statement is true for the current point in time.

Reply:
I'd say we're good as is but make sure it's there before we merge into master

Reply:
Ok, I've created a JIRA task for it